doc/ale-java.txt

```diff
@@ -114,8 +114,8 @@ Note: currently, the build can only run when launched with JDK 8. JDK 9 or more
 recent versions can be used to run the server though.
 
 After build completes the files required to run the language server will be
-located inside the folder `org.eclipse.jdt.ls.product/target/repository`.
-Ensure to set |g:ale_java_eclipselsp_path| to the absolute path of that folder.
+located inside the repositoy folder `eclipse.jdt.ls`. Ensure to set
```

Comment:
Typo here.

```suggestion
located inside the repository folder `eclipse.jdt.ls`. Please ensure you set
```

"Ensure to set" sounds weird, so I suggested "Please ensure you set."

Reply:
Fixed.